[
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1505144165.000070",
        "text": "<https:\/\/betfairlightweight.slack.com\/archives\/C4HL6EZTQ\/p1497360179025752> <@U4H3EEV45> did you ever isolate this error? I think I am seeing the same thing at least once a week ",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "attachments": [
            {
                "from_url": "https:\/\/betfairlightweight.slack.com\/archives\/C4HL6EZTQ\/p1497360179025752",
                "fallback": "[June 13th, 2017 6:22 AM] agberk: I'm also going to try setting the field on the socket `suppress_ragged_eofs` to False and run my code - if that's the issue then I should expect to see these exceptions get raised in my logs following by an automatic restart which my code takes care of",
                "ts": "1497360179.025752",
                "author_id": "U4H3EEV45",
                "author_subname": "agberk",
                "channel_id": "C4HL6EZTQ",
                "is_msg_unfurl": true,
                "text": "I'm also going to try setting the field on the socket `suppress_ragged_eofs` to False and run my code - if that's the issue then I should expect to see these exceptions get raised in my logs following by an automatic restart which my code takes care of",
                "author_name": "Aaron Berk",
                "author_link": "https:\/\/betfairlightweight.slack.com\/team\/agberk",
                "author_icon": "https:\/\/secure.gravatar.com\/avatar\/ea8143e60fb603cad63a14bfd181f72d.jpg?s=48&d=https%3A%2F%2Fa.slack-edge.com%2F66f9%2Fimg%2Favatars%2Fava_0025-48.png",
                "mrkdwn_in": [
                    "text"
                ],
                "id": 1,
                "footer": "Slack conversation"
            }
        ],
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "DoG",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "link",
                                "url": "https:\/\/betfairlightweight.slack.com\/archives\/C4HL6EZTQ\/p1497360179025752"
                            },
                            {
                                "type": "text",
                                "text": " "
                            },
                            {
                                "type": "user",
                                "user_id": "U4H3EEV45"
                            },
                            {
                                "type": "text",
                                "text": " did you ever isolate this error? I think I am seeing the same thing at least once a week"
                            }
                        ]
                    }
                ]
            }
        ]
    }
]